tests/strategies/mapping/test_mapping.py

```diff
@@ -0,0 +1,51 @@
+"""Test mapping strategy."""
```

Comment:
```suggestion
"""Test mapping semantic triples strategy."""
```

Reply:
I think this is simply a basic mapping strategy - not specially related to triples.